### ISSUE TITLE
For Thong

### ISSUE THREAD

**Issue by milailov2021** [milailov2021/chat#5]
Hi, Bro.

**Comment by milailov2021** [comment]
Please leave a msg if you have any news.

**Comment by thongtran8197** [comment]
yes, I recevied the money. I will contact again to Freelancer Supporter.

**Comment by thongtran8197** [comment]
hi

**Comment by milailov2021** [comment]
Please share the image again.

**Comment by thongtran8197** [comment]
Hi Thong! This is Dalton. How may I assist you today?    
15:17
I have 1 one project locked money. Yesterday, Freelancer Support help me send email to client to resolve it, but currently my money is still locked. Could you help me verify the progress resolving the issues.


15:19
My projects ID: #38607469

15:20

Thong Tran attached this photo

Click here to see the full image

15:20
Thank you for providing your concern.
15:21
Please note that we have already reached out to your client regarding this.
15:21
This is to remind them to resolve their account issues.

15:21
Once settled on their end, the funds will be unlocked.
15:21
1 question: In case my client can not resolve their account issues, means i will lose the money.


15:25
any rule to protect the freelancer?

15:25
You can reach back to us again after 30-days if they remain uncooperative so our team can review and provide options for you.

15:26
okay, i will wait 

15:26
Thank you Dalton

**Comment by thongtran8197** [comment]
means we have wait at least 30days or the clients must resolve accounts issues

**Comment by milailov2021** [comment]
omg, hope client to come back.

**Comment by thongtran8197** [comment]
seem the client or freelancer page is cheater.

**Comment by milailov2021** [comment]
the rule of freelancer !!!!

**Comment by milailov2021** [comment]
anyway the client paid money, and the money remains on my side in locked state. In other words, the money stays in freelancer.

**Comment by thongtran8197** [comment]
in meantime, If you have any client that want to pay you the money. Use my paypal, after  i will send to crypto wallet for you.

**Comment by thongtran8197** [comment]
Freelancer is dirty page.

**Comment by milailov2021** [comment]
Yes, really dirty.

**Comment by milailov2021** [comment]
And if they check that the developer shared the contact info outside of freelancer, then they close the dev's account.

**Comment by thongtran8197** [comment]
got it

**Comment by milailov2021** [comment]
Thanks for your effort.

**Comment by milailov2021** [comment]
Bro, is there any way to find him using his this info?

**Comment by thongtran8197** [comment]
If he does this on purpose, we can only wait. I saw the address you gave him and he is in the US

**Comment by thongtran8197** [comment]
I wonder if that money is being held by the Freelancer or him.

**Comment by milailov2021** [comment]
The money is being held by the Freelancer, hence it didn't this on purpose, in other words he can help me.

**Comment by thongtran8197** [comment]
I created the withdraw. Let continue your working.

**Comment by thongtran8197** [comment]
hi, If the withdraw success, I will send 350 to your wallet.

**Comment by milailov2021** [comment]
and will send you another wallet to send me 150 more.

**Comment by thongtran8197** [comment]
hi

**Comment by milailov2021** [comment]
here

**Comment by thongtran8197** [comment]
i sent and am waiting the process complete or not. If not, i will try trc20 address.

**Comment by milailov2021** [comment]
ok

**Comment by milailov2021** [comment]
to which account are you sending?

**Comment by thongtran8197** [comment]
![IMG_0675](https://github.com/user-attachments/assets/ba40720b-cc6f-4991-9da8-da2263c79f2a)
![IMG_0674](https://github.com/user-attachments/assets/d5707ad6-9708-468c-97d2-f12bfb4d3360)

**Comment by thongtran8197** [comment]
down them and i will remove.

**Comment by thongtran8197** [comment]
here?

**Comment by milailov2021** [comment]
Thx, one minute will down.

**Comment by milailov2021** [comment]
done, Thanks for your effort.

**Comment by thongtran8197** [comment]
thanks. i am outage now

**Comment by milailov2021** [comment]
Good work, Thanks again. See you.

**Comment by thongtran8197** [comment]
see you.

**Comment by thongtran8197** [comment]
Yes, i changed password. Do you want use it?

**Comment by milailov2021** [comment]
please share info.

**Comment by thongtran8197** [comment]
desryb-bisse3-wYkvec

**Comment by milailov2021** [comment]
Thanks.

**Comment by thongtran8197** [comment]
Download and remove it
